docs/pushnotificationios.md

```diff
@@ -12,97 +12,93 @@ title: '🚧 PushNotificationIOS'
   </p>
 </div>
 
-Handle push notifications for your app, including permission handling and icon badge number.
+Handle notifications for your app, including scheduling and permissions.
 
-To get up and running, [configure your notifications with Apple](https://developer.apple.com/library/ios/documentation/IDEs/Conceptual/AppDistributionGuide/AddingCapabilities/AddingCapabilities.html#//apple_ref/doc/uid/TP40012582-CH26-SW6) and your server-side system.
+## Getting Started
 
-React Native version equal or higher than 0.60.0:
+To enable push notifications, [configure your notifications with Apple](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server) and your server-side system.
 
-- Autolinking in 0.60.0 handles the linking for you!
-
-React Native versions lower than 0.60.0:
-
-Add the PushNotificationIOS library to your Podfile: ./ios/Podfile
+Then enable Background Modes/Remote notifications to be able to use remote notifications properly. The easiest way to do this is via the project settings. Navigate to Targets -> Your App -> Capabilities -> Background Modes and check Remote notifications. This will automatically enable the required settings.
```

Comment:
i think this can be explained better.

essentially, if your app is backgrounded and you want it to send it a silent notification that won't alert, you need to enable this setting.